starter-code/src/functions-and-arrays.js

```diff
@@ -11,10 +24,47 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord(words) {
+  var longestWord = '';
```

Comment:
This function will fail returning undefined when the array is empty. You can solve this checking if the array is empty and returning undefined then. A better way is asign to your comparator (longestWord) the first array's element. If the array is empty, longestWord will be 'undefined' and the function will return 'undefined'. Using an internal element for comparision solve some tricky problems (e.g. finding a minimum).
```
function findLongestWord(words) {
  var longestWord = words[0];

  for (var i = 1; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }
  return longestWord;
}
```